build.gradle

```diff
@@ -317,7 +330,7 @@ model {
                     lib library: "aes", linkage: "static"
 
                     exportedHeaders {
-                        srcDir generatedNativeHeadersDir
+                        srcDirs generatedNativeHeadersDir
```

Comment:
Is it necessary? `srcDir` is a valid method according to [docs](https://docs.gradle.org/current/dsl/org.gradle.api.file.SourceDirectorySet.html#org.gradle.api.file.SourceDirectorySet:srcDir(java.lang.Object))